backend/danswer/datastores/vespa/store.py

```diff
@@ -183,12 +152,10 @@ def _index_vespa_chunks(
             METADATA: json.dumps(document.metadata),
             EMBEDDINGS: embeddings_name_vector_map,
             BOOST: DEFAULT_BOOST,
-            ALLOWED_USERS: cross_connector_document_metadata_map[
-                document.id
-            ].allowed_users,
-            ALLOWED_GROUPS: cross_connector_document_metadata_map[
-                document.id
-            ].allowed_user_groups,
+            # give each document set "equal weighting" in the weightedset
```

Comment:
Is this for upcoming feature of document-sets boosting?

If this is supported, how will we handle different personas or knowledge sets favoring different document sets?

Reply:
for future clarity, let's call them personas which have document sets (knowledge sets as a term is banned!)